src/monitoringLambda/MessageFormatter.ts

```diff
@@ -1,31 +1,43 @@
 import { CloudWatchLogsDecodedData } from 'aws-lambda';
+import { Message } from './Message';
 import { SlackMessage } from './SlackMessage';
 import { getEventType } from './SnsEventHandler';
 
 /**
  * Abstract class for formatting differnt types of events
- * the formattedMesssage method returns a SlackMessage object
+ * the formattedMesssage method returns a Message object
  */
 export abstract class MessageFormatter<T> {
 
   event: T;
   account: string;
+  priority: string;
 
-  constructor(event: T, account: string) {
+  constructor(event: T, account: string, priority: string) {
     this.event = event;
     this.account = account;
+    this.priority = priority;
   }
 
   formattedMessage(): SlackMessage {
-    const message = new SlackMessage();
-    return this.constructMessage(message);
+    const message = new Message();
+    this.constructMessage(message);
+    return this.addMessageInteractions(message);
   }
 
-  abstract constructMessage(message: SlackMessage): SlackMessage;
+  abstract constructMessage(message: Message): Message;
+
+  addMessageInteractions(message: Message) {
```

Comment:
Vooral dit stukje is belangrijk. Hier wordt het topdesk incident toegevoegd aan elk geformateerde bericht.